src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -181,9 +181,10 @@ def init_db():
     Connects to DB
     :return: db_conn
     """
+    SonicDBConfig.load_sonic_global_db_config()
     # SyncD database connector. THIS MUST BE INITIALIZED ON A PER-THREAD BASIS.
     # Redis PubSub objects (such as those within swsssdk) are NOT thread-safe.
-    db_conn = SonicV2Connector(**redis_kwargs)
```

Comment:
>redis_kwargs [](start = 33, length = 12)

Why remove existing feature?

Reply:
This change is still arguable and irrelevant to the LLDP test fix. Could you minimize the changes in this PR, and submit another PR if you really want to change this part?

---
In reply to: [503657219](https://github.com/Azure/sonic-snmpagent/pull/166#discussion_r503657219) [](ancestors = 503657219)

Reply:
reverted back as suggested, only to provide fix for unit-test.